packages/office-ui-fabric-react/src/components/CommandBar/examples/CommandBar.ButtonAs.Example.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+
+import { CommandBar, ICommandBarProps } from 'office-ui-fabric-react/lib/CommandBar';
+import { CommandBarButton, IButtonProps } from 'office-ui-fabric-react/lib/Button';
+
+
+export class CommandBarButtonAsExample extends React.Component<ICommandBarProps, {}> {
+
+  constructor(props: ICommandBarProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+
+    const customButton = (props: IButtonProps) =>
+      <CommandBarButton
+        onMouseEnter={ () => console.log(`${props.text} hovered`) }
```

Comment:
Even though this is but just an example, we should probably still adhere to the documented best practices and avoid inline functions.